docs/user/exercises/test-case-best-practices.inc

```diff
@@ -0,0 +1,406 @@
+General best practices
+^^^^^^^^^^^^^^^^^^^^^^
+
+- **Write meaningful comments for your tests**
+
+  These comments should contain information about what is tested specifically, which task from the problem statement
+  is addressed or which TODO (if there are numbered TODOs in the template), how many points this test is worth when
+  passed and more if necessary.
+
+  .. code-block:: java
+
+    /**
+     * Tests that borrow() in Book successfully sets the available attribute to false
+     * Problem Statement Task 2.1
+     * Worth 1.5 Points (Weight: 1)
+     */
+    @Test
+    public void testBorrowInBook() {
+        // Test Code
+    }
+
+  Better yet, for manual correction, use these comments in the display name of the test. This allows the assessors, who execute the
+  tests in the IDE, to have more meaningful names displayed. The following example would make counting points easier.
+
+  .. code-block:: java
+
+    @DisplayName("1.5 P | Books can be borrowed successfully")
+    @Test
+    public void testBorrowInBook() {
+        // Test Code
+    }
+
+- **Use appropriate and descriptive names for test cases**
+
+  After exercises and exams, test names will be used to create statistics. If the tests are called *test1*, *test2*, *test3*,
+  it will be hard to read those statistics. This is the same reason as to why you should not name your variables ``int a``,
+  ``double b``, ``String c``. For example, if you want to test the method ``borrow`` in the class ``Book``,
+  ``testBorrowInBook()`` would be an appropriate name for the test case.
+
+  .. code-block:: java
+
+    @Test
+    public void testBorrowInBook() {
+        // Test Code
+    }
+
+  If you have many tests in different (nested) classes that are not completely distinct, add the name of the tested
+  class to the test to avoid having two tests with the same names. For example, if you test both ``add`` methods of a
+  LinkedList and an ArrayList. Clear and distinct test names also makes it easier to read and configure the grading in Artemis.
```

Comment:
Maybe make it clearer that this would lead to errors and not only some inconvenience on Artemis